internal/xds/cache/snapshotcache.go

```diff
@@ -25,14 +24,19 @@ import (
 	discoveryv3 "github.com/envoyproxy/go-control-plane/envoy/service/discovery/v3"
 	cachev3 "github.com/envoyproxy/go-control-plane/pkg/cache/v3"
 	serverv3 "github.com/envoyproxy/go-control-plane/pkg/server/v3"
+	"go.opentelemetry.io/otel"
+	"go.opentelemetry.io/otel/trace"
 	"go.uber.org/zap"
 
 	"github.com/envoyproxy/gateway/internal/logging"
 	"github.com/envoyproxy/gateway/internal/metrics"
 	"github.com/envoyproxy/gateway/internal/xds/types"
 )
 
-var Hash = cachev3.IDHash{}
+var (
+	Hash   = cachev3.IDHash{}
+	tracer = otel.Tracer("envoy-gateway/gateway-api")
```

Comment:
```suggestion
	tracer = otel.Tracer("envoy-gateway/xds")
```

Reply:
also does the `envoy-gateway` prefix exist since thats the idiomatic way to do things ?